<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>391</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>541</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>638</width>
    <height>177777</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>形状学习器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>学  习</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGraphicsView" name="graphicsViewStu">
          <property name="minimumSize">
           <size>
            <width>492</width>
            <height>166</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16666666</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="StugroupBox">
            <property name="title">
             <string>形状</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="StuButtonAgain">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>再来一个</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>测  试</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="3">
         <widget class="QGraphicsView" name="graphicsViewTest">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>492</width>
            <height>166</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16666</width>
            <height>16666</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="TestBoxAnswer">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>答题</string>
          </property>
          <widget class="QRadioButton" name="TestRadioNone">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>20</y>
             <width>47</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>操作</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="TestStart">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>199999</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>开始答题</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="TestKeys">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>1999999</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>回看正确答案</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="TestSubmit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>提交</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>354</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>查看历史</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>基本设置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
